core/src/Cuda/Kokkos_CudaSpace.cpp

```diff
@@ -51,6 +51,15 @@
 #include <algorithm>
 #include <atomic>
 
+// The purpose of the following variable is to allow a state-based choice
+// for pinning UVM allocations to the CPU. For now this is considered
+// an experimental debugging capability - with the potential to work around
+// some CUDA issues.
+#ifdef KOKKOS_IMPL_DEBUG_CUDA_PIN_UVM_TO_HOST
+bool kokkos_impl_cuda_pin_uvm_to_host_v = false;
+#define KOKKOS_IMPL_DEFINED_CUDA_PIN_UVM_TO_HOST_V
```

Comment:
```suggestion
```

Reply:
I think those ones are not needed at all anymore